examples/crd/pkg/resource.go

```diff
@@ -62,61 +56,6 @@ type ChaosPodList struct {
 	Items           []ChaosPod `json:"items"`
 }
 
-// +kubebuilder:webhook:path=/validate-chaosapps-metamagical-io-v1-chaospod,mutating=false,failurePolicy=fail,groups=chaosapps.metamagical.io,resources=chaospods,verbs=create;update,versions=v1,name=vchaospod.kb.io
-
-var _ webhook.Validator = &ChaosPod{}
-
-// ValidateCreate implements webhookutil.validator so a webhook will be registered for the type
-func (c *ChaosPod) ValidateCreate() (admission.Warnings, error) {
-	log.Info("validate create", "name", c.Name)
-
-	if c.Spec.NextStop.Before(&metav1.Time{Time: time.Now()}) {
-		return nil, fmt.Errorf(".spec.nextStop must be later than current time")
-	}
-	return nil, nil
-}
```

Comment:
Can we add an example with the CustomValidator interface?

Reply:
There are existing examples in https://github.com/kubernetes-sigs/controller-runtime/tree/main/examples/builtins